Detectors/FOCAL/simulation/src/Detector.cxx

```diff
@@ -261,14 +395,347 @@ AliMixture( 8, "Ceramic    $", acer, zcer, denscer, 2, wcer);
          isxfld, sxmgmx, 10.0, 1.0, 0.1, epsil, 0.001, nullptr, 0);
 }
 
+//____________________________________________________________________________
+void Detector::addAlignableVolumes() const
+{
+  // Create entries for alignable volumes associating the symbolic volume
+  // name with the corresponding volume path. Needs to be syncronized with
+  // eventual changes in the geometry
+  // Alignable volumes are:
+
+  // AddAlignableVolumesECAL();
+  addAlignableVolumesHCAL();
+}
+
+//____________________________________________________________________________
+void Detector::addAlignableVolumesHCAL() const
+{
+  TString vpsector = "/cave_1/barrel_1/FOCAL_1/HCAL_1";
```

Comment:
FOCAL is a forward detector, looks strange to me to put it into the volume `barrel_1`. I think FOCAL_1 should be a separate volume directly under cave.

Reply:
FOCAL has been moved to caveRB24.